### PR TITLE
Footer

### PR DESCRIPTION
**Pull request by TykhaA**
footer mobile

### CODE REVIEW

File: src/css/footer.css [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+  font-weight: 600;
+}
+
+.footer-nav {
+  margin-bottom: 40px;
+}
+
+.footer-list-menu .footer-item {
+  position: relative;
+  transition: opacity 250ms ease;
```
Comment (asaulyak) [inline src/css/footer.css:68]: use `var(--animation-fast)`

File: src/css/footer.css [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+}
+
+.footer-social-item {
+  border-radius: 10px;
+  border: 1px solid rgba(255, 255, 255, 0.1);
+  padding: 8px;
+  display: flex;
+  align-items: center;
+  cursor: pointer;
+  transition: border-color 250ms cubic-bezier(0.075, 0.82, 0.165, 1);
```
Comment (asaulyak) [inline src/css/footer.css:99]: use `var(--animation-fast)`

File: src/css/footer.css [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+}
+
+.footer-social-item:hover,
+.footer-social-item:focus {
+  border: 1px solid rgba(77, 194, 116, 0.1);
```
Comment (asaulyak) [inline src/css/footer.css:104]: add color to the vars or use the existing one

File: src/css/footer.css [hunk 4]
```diff
@@ -0,0 +1,116 @@
...
+}
+
+.footer-icon {
+  fill: var(--white);
+  opacity: 0.8;
+  transition: fill 250ms cubic-bezier(0.075, 0.82, 0.165, 1);
```
Comment (asaulyak) [inline src/css/footer.css:110]: use `var(--animation-fast)`

File: src/partials/footer.html [hunk 5]
```diff
@@ -1 +1,82 @@
...
+          </li>
+          <li class="footer-list-item">
+            <a href="" class="footer-item">Contact us</a>
+          </li>
+        </ul>
+      </nav>
+      <ul class="footer-social">
+        <li>
+          <a
+            href="https://www.facebook.com/goITclub/"
```
Comment (asaulyak) [inline src/partials/footer.html:43]: add `target="_blank" rel="noopener noreferrer"` to the external links

File: src/partials/footer.html [hunk 6]
```diff
@@ -1 +1,82 @@
...
+            </svg>
+          </a>
+        </li>
+      </ul>
+    </div>
+    <div class="footer-bottom">
+      <p class="text">&#169;2023 English Excellence. All rights reserved.</p>
+      <ul class="footer-bottom-list">
+        <li class="footer-bottom-item">
+          <a class="link" href="" target="blank">Privacy Policy</a>
```
Comment (asaulyak) [inline src/partials/footer.html:74]: add `rel="noopener noreferrer"`

File: src/partials/footer.html [hunk 7]
```diff
@@ -1 +1,82 @@
...
+        </li>
+        <li class="footer-bottom-item">
+          <a class="link" href="" target="blank">Terms of Service</a>
```
Comment (asaulyak) [inline src/partials/footer.html:77]: add `rel="noopener noreferrer"`

File: src/css/main.css [hunk 8]
```diff
@@ -89,6 +89,7 @@
 
   transition: background-color var(--animation-fast);
 }
+
 .button:hover {
   background-color: #38ae5f;
```
Comment (asaulyak) [inline src/css/main.css:94]: use variable for color

**Review by asaulyak — COMMENTED** [review]
footer should have multiple columns on desktop and tablet

File: src/partials/footer.html [hunk 9]
```diff
@@ -1,39 +1,46 @@
...
+                >+380 (68) 443-94-26</a
+              >
+            </li>
+            <li class="footer-list-item">
+              <p class="item-title">Address:</p>
+              <a
+                href="https://maps.app.goo.gl/bPwTf8E8XLwf26PZ7"
+                target="_blank"
+                class="footer-item"
+                >32a M. Bazhana ave</a
```
Comment (asaulyak) [inline src/partials/footer.html:22]: rel is missing

File: src/css/footer.css [hunk 10]
```diff
@@ -4,19 +4,58 @@
...
+
+@media only screen and (min-width: 768px) {
+  .footer-top {
+    display: flex;
+    justify-content: space-between;
+  }
+}
+
 .footer-bottom {
   color: rgba(255, 255, 255, 0.6);
```
Comment (asaulyak) [inline src/css/footer.css:43]: use var color

File: src/css/footer.css [hunk 11]
```diff
@@ -65,38 +120,56 @@
...
 .footer-list-menu .footer-item {
   position: relative;
-  transition: opacity 250ms ease;
 }
 
 .footer-list-menu .footer-item:after {
   position: absolute;
   content: '';
-  width: 37px;
+  width: 0px;
```
Comment (asaulyak) [inline src/css/footer.css:128]: why width: 0?

File: src/css/footer.css [hunk 12]
```diff
@@ -65,38 +120,56 @@
...
   height: 2px;
   left: 0;
   bottom: -5px;
   background-color: var(--green);
   opacity: 0;
+  transition:
+    opacity 250ms ease,
```
Comment (asaulyak) [inline src/css/footer.css:135]: use var(--animation-fast)

File: src/css/footer.css (hunk 11 above)
Comment (TykhaA) [reply src/css/footer.css:128]: for animation to showing on hover from left to right

**Review by asaulyak — APPROVED** [review]

**Pull request merged**